scripts/run_integration_test.sh

```
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Script to locally run the functional integration test.
+#
+
```

we probably want to also do `set -e` to fail fast upon internal bash errors